src/app/projects/project-details/project-details-build/project-details-build.component.ts

```diff
@@ -86,22 +88,30 @@ export class ProjectDetailsBuildComponent {
     }
   }
 
+  numberWithCommas(num: number) {
+    return this.numberFormat.format(num);
+  }
+
   private fillProjectBuilds(proj: WharfProject): Observable<WharfProject> {
     return forkJoin(
-      proj.buildHistory.map(x =>
-        this.artifactService
-          .buildBuildidTestsResultsGet(x.buildId)
-          .pipe(
-            map(
-              testResults =>
-                new TestsResults(testResults.passed, testResults.failed),
+      proj.buildHistory.map(x => this.testResultService
+        .buildBuildidTestResultListSummaryGet(x.buildId)
+        .pipe(
+          map(testResultListSummary =>
+            new TestResultListSummary(
+              testResultListSummary.buildId,
+              testResultListSummary.failed,
+              testResultListSummary.passed,
+              testResultListSummary.skipped,
+              testResultListSummary.total,
             ),
           ),
+        ),
```

Comment:
This still produces the "N+1" issue we have with the build list. Does not the "get project builds" endpoint return the test result list summary? That was one of the major arguments for at least half of the refactor